persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/ReflectionUtils.java

```diff
@@ -84,10 +79,6 @@
     }
   }
 
-  static Response execute(Request request, QueryState queryState, long queryStartNanoTime) {
```

Comment:
Looks like they did change this to public
https://github.com/apache/cassandra/commit/a7c4ba9eeecb365e7c4753d8eaab747edd9a632a#diff-8c4c40abb1aac546629cbf00371d0f93bac1de96d20f064bec2b31968b2e9a71R219

Reply:
Yep..